recipes/apsg/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "apsg" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d2c0b8f68fa3dc68e44d6ee83e5b01d894b1f2567f89fc97ff00749528dcef06
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
+  entry_points:
+    - iapsg=apsg.shell:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - apsg
```

Comment:
can you add command test which tests the CLI?

Reply:
It is not really CLI app. CLI is used just to open an interactive shell and import APSG. I'm not sure how to test it.

Reply:
Is there a `--help` option?

Reply:
No, there are no command line arguments implemented. The interactive shell is still experimental and could be improved in next versions, but now it is sort of undocumented feature.